src/Main.elm

```diff
@@ -281,14 +321,23 @@ decimalStatusToString status =
             "NoDecimal"
 
 
-actionToString : Action -> String
+actionToString : Maybe Action -> String
```

Comment:
Why would `actionToString` take a `Maybe Action`? Surely there must be some way to avoid this oddity.

Reply:
Changed the name to maybeActionToString. This is just used in my testing, to display what action is saved. This will be committed in the next PR.